go/noise/laplace_noise.go

```diff
@@ -164,14 +165,30 @@ func (laplace) DeltaForThreshold(l0Sensitivity int64, lInfSensitivity, epsilon,
 	return 1 - math.Pow(1-partitionDelta, float64(l0Sensitivity))
 }
 
+// ConfidenceIntervalInt64 computes a confidence interval that contains int64 noisedValue with probablility
+// greater or equal to the given confidenceLevel with the given noise parameters.
 func (laplace) ConfidenceIntervalInt64(noisedValue, l0Sensitivity, lInfSensitivity int64, epsilon, delta, confidenceLevel float64) (ConfidenceInterval, error) {
-	// TODO: Implement confidence interval computation.
-	return ConfidenceInterval{}, nil
+	if err := checkArgsConfidenceIntervalLaplace("ConfidenceIntervalInt64 (Laplace)", l0Sensitivity, float64(lInfSensitivity), epsilon,
+		delta, confidenceLevel); err != nil {
+		err = fmt.Errorf("ConfidenceIntervalInt64(noisedValue %d, l0sensitivity %d, lInfSensitivity %d, epsilon %f,"+
+			"delta %e, confidenceLevel %f) checks failed with %v", noisedValue, l0Sensitivity, lInfSensitivity, epsilon, delta, confidenceLevel, err)
+		return ConfidenceInterval{}, err
+	}
+	lambda := laplaceLambda(l0Sensitivity, float64(lInfSensitivity), epsilon)
+	return getConfidenceIntervalLaplace(float64(noisedValue), lambda, confidenceLevel).toConfidenceIntervalInt64(), nil
```

Comment:
Same as above. It might make sense to keep int64 since round does round to integer representable in 64 bits?
https://golang.org/src/math/floor.go?s=1237:1266#L54 not sure though.

Reply:
I also argued about renaming it to roundConfidenceInterval, but the reasoning of not renaming it was, like Karim said, the fact that we "represent" a float64 as an int64.

Reply:
This was changed with the last commit to https://github.com/google/differential-privacy/blob/1b1dc6639173c0a13613189ec21851604a4c7335/go/noise/noise.go#L69 "toInt64()".